packages/gatsby-source-drupal/src/__tests__/fixtures/schema.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Tihs file seems to be indented by 4 spaces. The other json file has 2 spaces. I actually don't know what we set json files to but the inconsistency here makes me wonder whether Prettier ran at all :)